code/game/machinery/doors/poddoor/poddoor.dm

```diff
@@ -0,0 +1,118 @@
+
+/obj/structure/machinery/door/poddoor
```

Comment:
Empty line here. (It's there in the original file too, but may as well remove it in this)
```suggestion
/obj/structure/machinery/door/poddoor
```